src/utils/OpenSwathWorkflow.cpp

```diff
@@ -416,8 +417,8 @@ class TOPPOpenSwathWorkflow
     registerInputFile_("swath_windows_file", "<file>", "", "Optional, tab separated file containing the SWATH windows for extraction: lower_offset upper_offset \\newline 400 425 \\newline ... Note that the first line is a header and will be skipped.", false, true);
     registerFlag_("sort_swath_maps", "Sort of input SWATH files when matching to SWATH windows from swath_windows_file", true);
 
-    registerFlag_("use_ms1_traces", "Extract the precursor ion trace(s) and use for scoring", true);
-    registerFlag_("enable_uis_scoring", "Enable additional scoring of identification assays", true);
+    registerFlag_("enable_ms1", "MS1: Set this flag if precursor ion trace(s) should be extracted and used for scoring", false);
```

Comment:
@hroest Do you think this name change makes sense or do you prefer the old (or a different) name?

Reply:
well why not just make it the default option and  use `registerStringOption_("enable_ms1" ... ` ?

Reply:
That's a good suggestion. If no MS1 maps are there, I believe this will be skipped anyway, right? I will make the same changes for ``enable_ipf`` as well, because this will run only if identification transitions are present in the library.